docs/operational_guide/kernel_configuration.md

```diff
@@ -31,12 +31,29 @@ To set this value permanently, update the `vm.swappiness` setting in `/etc/sysct
 ## rlimits
 M3DB also can use a high number of files and we suggest setting a high max open number of files due to per partition fileset volumes.
 
-On Linux you can set a high limit for maximum number of open files in `/etc/security/limits.conf`:
+On Linux you can set a high limit for the maximum number of open files for a specific user in `/etc/security/limits.conf`:
```

Comment:
We might want to mention that when running under systemd this vals will probably be ignored and maybe point to some of the GH issues or docs we found. Let me know if you can't find some refs I can help look

Reply:
I think a comment should be enough and then tell them to see the systemd details below

Reply:
`/etc/security/limits.conf` only works for PAM stuff, let's not mislead users. It must be set in systemd unit, upstart config or the init script itself.